CHANGELOG.md

```diff
@@ -5,6 +5,20 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 0.4.0 - 2024-02-27
+
+### Removed
+
+- The `Applicant` entity no longer exists.
+- All `/applicants/*` endpoints are removed.
+- The `applicantId` attribute of `Proposal` has been removed.
+
+### Added
+
+- The `Organization` entity now exists.
+- The `/organizations` POST and GET endpoint now exists.
+- The `/organizations/{organizationId}` endpoint now exists.
+
```

Comment:
Could you move the organization stuff to the other PR?

Reply:
Actually… I bet the reason you saved all of it for this PR was so you could merge the org one in without triggering a release, then release it all as 0.4.0? I wonder if you should just do something like add a `0.4.0 (under development)` to the CHANGELOG (without actually changing the version number in the package) in the org PR, then in this PR remove the `(under development)`, add the applicant items, and do the real version bumping? 🤔

Reply:
Meh.  This whole thing is weird and strange and uncomfortable.  I made a commit in the other PR but realized from a git history PoV it wasn't any better (sure, it'd be in the other PR but it was still decoupled from the commit changes themselves).

On the flip side, the changes were made over 3 commits, so I'd have to do three "releases" which is silly...  or use "unreleased"

Like I said, the concept of unreleased is a lie: things are released once they're merged.

Maybe the mistake here is thinking of "releases" at all -- maybe we should just have a change log with dates, and we could put the current swagger spec version next to every change.  For right now I'm going to do the "single commit with a `0.3.1` "release" even though that's silly, but I think going forward we should flatten the changelog and have it be date based with "current API version" as metadata.

Reply:
I'm fine with whatever solution you come to this, tbh! It was a minor complaint and I should've marked it as optional / up to your judgement. That said:

> Like I said, the concept of unreleased is a lie: things are released once they're merged.

Not to be pedantic, but just to make sure I know the current state of things: I think this is the correct way of thinking about it, but technically we neither deploy automatically nor release automatically (unless version number is incremented), right? Or no? (Either way you're right about living by that edict regardless of whether it's the ground truth atm.)

Reply:
I wonder if the way to solve this is _actually_ to merge this PR into #794, and then merge that one in? I know we don't normally do that, but… then we get just a single release to `main`. Is… is this a bad pattern?

Reply:
I'd prefer to not do that pattern (and would not want that to become a standard pattern) -- ultimately we need to be able to merge things to main.  Having a "dev" branch isn't what we do (and we don't want one, nor do we want every-branch-becomes-a-dev-branch).

Appreciated that we don't truly have CD right now but that's more of a bug than a feature (and I think #622 is meant to fix it)